### PR TITLE
GDB-9464: Abort non-updateable queries when leaving the SPARQL view.

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
Added functionality to abort non-updateable queries when a tab or tabs is/are closed or when leaving the SPARQL view

## Why
When the yasgui component or a tab is destroyed and there is/are ongoing queries, the results of the queries won't be processed, which is why it's better to abort them.

## How
The version of "ontotext-yasgui-web-component" has been increased. The functionality of aborting the count queries is extended to abort non-updateable queries as well.

### CODE REVIEW

File: src/i18n/locale-en.json [hunk 1]
```diff
@@ -799,6 +799,26 @@
     "query.editor.automatically.execute.update.warning": "This is an update and it may change the data in the repository.<br>Are you sure you want to execute it automatically?",
     "query.editor.error.show.full.message": "Show full exception message",
     "query.editor.error.show.less.message": "Show less exception message",
+    "yasqe.leave_page.run_queries.confirmation.none_queries_non_updates.message": "Are you sure that you want to exit?",
```
Comment (svilenvelikov) [inline src/i18n/locale-en.json:802]: This is not the correct place for these labels. There is the `view.sparql-editor` section where these have to go

File: src/css/lib/ontotext-yasgui-web-component.css [hunk 2]
```diff
@@ -321,6 +321,10 @@ yasgui-component .yasr_plugin_control label {
     margin-bottom: 0;
 }
 
+.runQueryConfirmation {
```
[the remark this answers is not in the document]
Comment (boyan-tonchev) [reply src/css/lib/ontotext-yasgui-web-component.css]: Done

[the remark this answers is not in the document]
Comment (boyan-tonchev) [reply src/css/lib/ontotext-yasgui-web-component.css]: Done

File: src/js/angular/models/ontotext-yasgui/ran-request-count-info.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export class RanRequestCountInfo {
```
[the remark this answers is not in the document]
Comment (boyan-tonchev) [reply src/js/angular/models/ontotext-yasgui/ran-request-count-info.js]: Done

File: src/i18n/locale-en.json (hunk 1 above)
Comment (boyan-tonchev) [reply src/i18n/locale-en.json]: Done

**Review by svilenvelikov — APPROVED** [review]

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**